apps/dashboard/src/app/(app)/team/~/[[...paths]]/page.tsx

```diff
@@ -0,0 +1,122 @@
+import { type Team, getTeams } from "@/api/team";
+import { GradientAvatar } from "@/components/blocks/Avatars/GradientAvatar";
+import { AppFooter } from "@/components/blocks/app-footer";
+import { DotsBackgroundPattern } from "@/components/ui/background-patterns";
+import { getThirdwebClient } from "@/constants/thirdweb.server";
+import { ChevronRightIcon, UsersIcon } from "lucide-react";
+import Link from "next/link";
+import { notFound, redirect } from "next/navigation";
+import { getAuthToken } from "../../../api/lib/getAuthToken";
+import { TeamPlanBadge } from "../../../components/TeamPlanBadge";
+import { TeamHeader } from "../../components/TeamHeader/team-header";
+
+export default async function Page(props: {
+  params: Promise<{
+    paths?: string[];
+  }>;
+  searchParams: Promise<Record<string, string | undefined | string[]>>;
+}) {
+  const params = await props.params;
```

Comment:
These can all run at the same time in a promise all

Reply:
updated